### PR TITLE
Make base code compatible with Symfony3

### PR DESCRIPTION
**Pull request by egeloen**
This PR makes the base code compatible with Symfony3. The best choice seems to relies on method presence check.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,22 +6,20 @@ php:
...
-cache:
-    directories:
-        - vendor
-        - $HOME/.composer/cache
+        - SYMFONY_PHPUNIT_BRIDGE_VERSION=2.7.*
 
 install:
     - composer self-update
     - composer require --no-update symfony/form:${SYMFONY_VERSION}
+    - composer require --dev --no-update symfony/phpunit-bridge:${SYMFONY_PHPUNIT_BRIDGE_VERSION}
```
Comment (xabbuh) [inline .travis.yml]: Why not adding this to the `composer.json`? It's also useful when running tests locally.

Comment (xabbuh) [inline .travis.yml]: Ah I see you did that. But what's the reason to run tests with different versions of the bridge?

File: src/Extension/AbstractOrderedExtension.php [hunk 2]
```diff
@@ -31,12 +32,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(array('position' => null));
+
+        if (method_exists('Symfony\Component\Form\AbstractType', 'configureOptions')) {
```
Comment (xabbuh) [inline src/Extension/AbstractOrderedExtension.php]: This should check presence of a method from the OptionsResolver component as installed versions of the Form and OptionsResolver components can differ.

Comment (xabbuh) [inline src/Extension/AbstractOrderedExtension.php]: Or alternatively just stick with the call from the `else` block.

File: .travis.yml (hunk 1 above)
Comment (egeloen) [inline .travis.yml]: Yes, it can be removed and I will do.

File: src/Extension/AbstractOrderedExtension.php (hunk 2 above)
Comment (egeloen) [inline src/Extension/AbstractOrderedExtension.php]: But then, what can I check in the OptionsResolver in order to avoid deprecated calls?

Comment (xabbuh) [inline src/Extension/AbstractOrderedExtension.php]: You can check for one of the methods added to the `OptionsResolver` class in 2.6: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/OptionsResolver/CHANGELOG.md

Comment (egeloen) [inline src/Extension/AbstractOrderedExtension.php]: Thanks, will do!

**Pull request merged**